.github/workflows/swan_pr_workflow.yml

```diff
@@ -0,0 +1,146 @@
+# GitHub action file for SwanHub SDK
+name: SwanHub SDK Application Workflow
+
+# Triggers the workflow on push or pull request
+on:
+  push: # Triggered on a push to any branch ( do not change )
```

Comment:
I personally don't think we should build the image on every single branch,  is there a way to make it so that we run it on Pull Requests? Im more used to these actions being mainly for Dev and Main branch

Reply:
agree, its unnecessary.
i made changes